frontend/src/pages/org/archived-item-qa/ui/text.ts

```diff
@@ -0,0 +1,105 @@
+import { msg } from "@lit/localize";
+import { html } from "lit";
+import { guard } from "lit/directives/guard.js";
+import { until } from "lit/directives/until.js";
+import { when } from "lit/directives/when.js";
+
+import type { ReplayData, TextPayload } from "../types";
+
+import { renderSpinner } from "./spinner";
+
+import { tw } from "@/utils/tailwind";
+
+const diffImport = import("diff");
+
+function renderDiff(
+  crawlText: TextPayload["text"],
+  qaText: TextPayload["text"],
+) {
+  return until(
+    diffImport.then(({ diffWords }) => {
+      const diff = diffWords(crawlText, qaText);
+
+      const addedText = tw`bg-red-100 text-red-700`;
+      const removedText = tw`bg-red-100 text-red-100`;
+
+      return html`
+        <div
+          class=${tw`flex-1 overflow-hidden whitespace-pre-line rounded-lg border-dashed p-4 first-of-type:border-r`}
```

Comment:
I found the text diff much easier to parse as a large block.  While not exactly easy to _read_, I found it was quicker to see related parts of the page that were differences as a large block of text instead of multiple lines where the relation between those elements was less clear to me.

Also less scrolling!  Over many pages this will compound into lots of time saved!

```suggestion
          class=${tw`flex-1 overflow-hidden rounded-lg border-dashed p-4 first-of-type:border-r`}
```

Reply:
Let's try it, as this may look better with the diff tool we're using